binding/proxy.go

```diff
@@ -23,29 +22,47 @@ func (h *Proxy) Create(r *api.Proxy) (err error) {
 func (h *Proxy) Get(id uint) (r *api.Proxy, err error) {
 	r = &api.Proxy{}
 	path := Path(api.ProxyRoot).Inject(Params{api.ID: id})
-	err = h.Client.Get(path, r)
+	err = h.client.Get(path, r)
 	return
 }
 
 //
 // List Proxies.
 func (h *Proxy) List() (list []api.Proxy, err error) {
 	list = []api.Proxy{}
-	err = h.Client.Get(api.ProxiesRoot, &list)
+	err = h.client.Get(api.ProxiesRoot, &list)
 	return
 }
 
 //
 // Update a Proxy.
 func (h *Proxy) Update(r *api.Proxy) (err error) {
 	path := Path(api.ProxyRoot).Inject(Params{api.ID: r.ID})
-	err = h.Client.Put(path, r)
+	err = h.client.Put(path, r)
 	return
 }
 
 //
 // Delete a Proxy.
 func (h *Proxy) Delete(id uint) (err error) {
-	err = h.Client.Delete(Path(api.ProxyRoot).Inject(Params{api.ID: id}))
+	err = h.client.Delete(Path(api.ProxyRoot).Inject(Params{api.ID: id}))
+	return
+}
+
+//
+// Find by Kind.
+// Returns nil when not found.
```

Comment:
hint: Find() copied from addon/.  Used by the addons and (tackle2-addon) but generally useful enough to be included in the binding.